index.js

```diff
@@ -105,6 +105,17 @@ WPiPlatform.prototype.configureAccessory = function(accessory) {
       });
   }
 
+  if (accessory.getService(Service.MotionSensor) && accessory.context.mode === "in") {
```

Comment:
I'm thinking (as per your comment) that mode isn't really required any more - so if this service is defined and .type==="MotionSensor", then that should be the logic test - so .mode can be replaced with the .type check.